package.json

```diff
@@ -49,6 +51,7 @@
     "react-test-renderer": "^15.6.1",
     "sinon": "^3.3.0"
   },
+  "proxy": "http://localhost:3001",
```

Comment:
Does this cause a problem with heroku?

Reply:
@bill-feng Humm... we don't see a problem on the review app now, so probably ok? We added this for dev convenience, but it needs to be removed later.

Reply:
@bill-feng @kumiko-haraguchi I think it's fine to leave as is, actually.

Reply:
We only need 1 proxy though!